Automattic/wp-calypso
# Jetpack Partner Portal: Add pages for listing and adding payment methods

This PR created two new partner portal contexts, `paymentMethodAdd` and `paymentMethodList` which load two new pages that will support listing and adding credit cards to the payment portal. These pages are feature flagged and should only be available in development and staging right now.

#### Changes proposed in this Pull Request
- Introduces the jetpack/partner-portal-payment feature flag
- Adds payment method list page (feature-flagged) to Jetpack Partner Portal
- Adds payment method `add` page (feature-flagged) to Jetpack Partner Portal
- Adds sidebar link (feature-flagged) to the sidebar in Jetpack Partner Portal
- Rearranges the sidebar icons per i2 designs

#### Testing instructions

- Load up the partner portal in jetpack-cloud-development.
- Ensure that the new sidebar menu item "Payment Method" is present.
- Ensure that when clicking the sidebar menu item you are directed to the "list" page.
- Ensure that when clicking the "Add credit card" button you are directed to the "add" page.
- Observe javascript console for errors and/or warnings.
